frontend/src/components/job/info.js

```diff
@@ -50,23 +54,30 @@ class JobInfo extends Component {
         return (
             <div className="tab-pane active">
                 <div className="content">
-                    <legend>Job Properties</legend>
-                    <dl className="dl-horizontal">
-                        {jobProps.map((jobProp) => {
-                            let jobPropValue = this.props.job[jobProp]
-                            if (Array.isArray(jobPropValue)) {
-                                jobPropValue = jobPropValue.join(', ')
-                            }
-
-                            return (
-                                <div key={jobProp}>
-                                    <dt>{jobProp}</dt>
-                                    <dd>{jobPropValue}</dd>
-                                </div>
-                            )
-                        }, this)}
-                    </dl>
-
+                    <div className="row">
+                        <div className="col-lg-6 col-md-6 col-sm-12 col-sx-12">
+                            <legend>Job Properties</legend>
+                            <dl className="dl-horizontal">
+                                {jobProps.map((jobProp) => {
+                                    let jobPropValue = this.props.job[jobProp]
+                                    if (Array.isArray(jobPropValue)) {
+                                        jobPropValue = jobPropValue.join(', ')
+                                    }
+
+                                    return (
+                                        <div key={jobProp}>
+                                            <dt>{jobProp}</dt>
+                                            <dd>{jobPropValue}</dd>
+                                        </div>
+                                    )
+                                }, this)}
+                            </dl>
+                        </div>
+                        <div className="col-lg-6 col-md-6 col-sm-12 col-sx-12">
+                            <legend>Meta Properties</legend>
+                            {this.collectMeta(this.props.job.Meta || {}, "wide")}
```

Comment:
Is the extra ```|| {}``` necessary? It's already checked in the function as well and I think .Meta is always there on the job object, although it may be empty.